tests/everflow/test_everflow_ipv6.py

```diff
@@ -27,6 +28,19 @@ class EverflowIPv6Tests(BaseEverflowTest):
 
     DEFAULT_SRC_IP = "ffbe:0225:7c6b:a982:d48b:230e:f271:0000"
     DEFAULT_DST_IP = "ffbe:0225:7c6b:a982:d48b:230e:f271:0001"
+    tx_port_ids = []
+
+    @pytest.fixture(scope='class',  autouse=True)
+    def setup_mirror_session_dest_ip_route(self, duthosts, rand_one_dut_hostname, tbinfo, setup_info, setup_mirror_session):
+
+        duthost = duthosts[rand_one_dut_hostname]
+        tx_port = setup_info["tor"]["dest_port"][0]
+        peer_ip, _ = everflow_utils.get_neighbor_info(duthost, tx_port, tbinfo)
+        everflow_utils.add_route(duthost, setup_mirror_session["session_prefixes"][0], peer_ip)
+        EverflowIPv6Tests.tx_port_ids = self._get_tx_port_id_list([setup_info["tor"]["dest_port_ptf_id"][0]])
```

Comment:
Is there anything preventing us from using `self` here?

Reply:
@daall Thanks Updated. When I made changes some time back there were some errors when using 'self' but looks like that may be of some other reason.